### DISCUSSION TITLE
How to add vim/vi aliases or symlinks?

### DISCUSSION THREAD

**Discussion by damccull** [nix-community/kickstart-nix.nvim#40] [Q&A]
In ubuntu and derivatives, and possibly others, a command `update-alternatives` is used to create symlinks for certain interchangeable programs to select your defaults and alias commands. For neovim, you can enable it as your default editor, and provide `vi` and `vim` as aliases for `nvim`. How can I do this with the kickstart-nix.nvim flake? Is it an option already in that I'm missing or is this something I'll need to figure out how to build in?

**Comment by damccull** [comment]
I see that this functionality is supposed to already exist in the mkNeovim.nix file. However, it's not operating on my system and I have not renamed the binary, indicating that the defaults should create the two symlinks, unless there's a config option I'm still missing. Any advice?

**Reply by mrcjkb** [reply]
Hey :wave: 

I've just pushed a fix: https://github.com/nix-community/kickstart-nix.nvim/commit/43effe21225747b8b81bbb5e63208c8c63c85780

**Comment by damccull** [comment]
This worked great. Thanks. Instantly fixed.
